lenskit-eval/src/main/java/org/lenskit/eval/traintest/recommend/RecommendEvalTask.java

```diff
@@ -303,13 +305,17 @@ public ConditionEvaluator createConditionEvaluator(AlgorithmInstance algorithm,
             return null;
         }
 
+        boolean useDetails = tlb != null; // writing requires details
```

Comment:
This doesn't make sense to me (comment not helpful). Under what conditions is tlb null or not?